src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -114,6 +114,39 @@ class SubmissionComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     }
   }
 
+  it should "create multiple unique submissions concurrently" in withDefaultTestDatabase {
```

Comment:
each submission is (should be) unique, what happens here when we submit concurrently?